lib/msal-browser/docs/requestresponseobject.md

```diff
@@ -14,3 +84,45 @@ AAD will only allow you to get consent for 3 resources at a time, although you c
 When the user makes a login request, you can pass in multiple resources and their corresponding scopes because AAD issues an idToken pre consenting those scopes. However acquireToken calls are valid only for one resource / multiple scopes. If you need to access multiple resources, please make separate acquireToken calls per resource.
 
 You can read more about scopes and permissions [here](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-permissions-and-consent).
+
+### Prompt Values
+
+When setting the prompt value, you must provide one of the following:
+
+| Prompt Value | Description |
+| ------------ | ----------- |
+| `"login"` | Presents a login prompt. |
+| `"select_account"` | Present a select account prompt on the login screen - used for multiple account scenarios |
+| `"consent"` | Present a consent screen. |
+| `"none"` | No prompt - exclusively used for silent flows. |
+
+## Response
+
+Either in `handleRedirectCallback` or as a result of the promise from `loginPopup`, `acquireTokenPopup` or `acquireTokenSilent`, MSAL will return a `TokenResponse.ts` object.
+```javascript
+const response = {
+    uniqueId: "",
+    tenantId: "",
+    scopes: [],
+    tokenType: "",
+    idToken: "",
+    idTokenClaims: {},
+    accessToken: "",
+    refreshToken: "",
+    expiresOn: new Date(),
+    account: new Account()
+};
+```
+
+| Parameter | Description | Format |
```

Comment:
one other thing, do we care about normalizing the readability of these markdown tables? like having the spaces between | always be the same size, etc., so it'd look more like:

```
| `uniqueId` | `oid` or `sub` claim from ID token. | string |
| `tenantId` | `tid` claim from ID token.          | string |
```

basically lined up with the longest row in each column

Reply:
We can do that incrementally. I wrote these with the intent of viewing them in browser, but if it is easier to format these raw docs then I am happy to put some time into it. There may be libs or plugins that do it for us as well.